Source/WebCore/Modules/notifications/Notification.h

```diff
@@ -81,6 +83,7 @@ class Notification final : public ThreadSafeRefCounted<Notification>, public Act
     const String& lang() const { return m_lang; }
     const String& tag() const { return m_tag; }
     const URL& icon() const { return m_icon; }
+    JSC::JSValue data(JSC::JSGlobalObject&);
```

Comment:
Maybe we could call this one dataForBindings()? (You can use `[ImplementedAs=dataForBindings]` in the IDL)

And then keep the other one as `data()`, instead of having `data()` & `notificationData()`

Reply:
I like this better, too.